### ISSUE TITLE
GameGuru MAX - Request - Reduce Size of Storyboard Title Bar

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4122]
MAX tends to have quite a few pieces of the UI that tend to be oversized, imo. One of them is the title bar for the Storyboard editor. Once someone gets into serious game development, the storyboard area is going to get very busy with many levels/maps and possibly lots of other nodes. As a result, we will need as much screen real estate as possible to keep as much as possible in view while working in the Storyboard editor. One way to free up a little more room is to reduce the size of the Storyboard editor's title bar. Currently, it's huge and takes up a lot of vertical space. Here's an example comparing it to the MAX Hub's title bar:

_MAX HUB title bar:_
![titlebar1](https://user-images.githubusercontent.com/36735507/214809540-20adae9f-6f1b-4ec1-9698-f3507ef20f27.png)

_MAX Storyboard Editor title bar:_
![titlebar2](https://user-images.githubusercontent.com/36735507/214809585-0402469b-07df-43a0-a968-e2c66c862668.png)

The Storyboard title bar appears to be at least twice as tall. Why not make this smaller. It serves no purpose other than to let the end-user know they are in the Storyboard Editor and that could easily be accomplished with a smaller title bar.

**Comment by davetgc** [comment]
Maybe even make it hideable?

**Comment by OldFlak** [comment]
Yes - tis just a waist of screen real-estate - I promise I know when I have invoked the story board. Could even just remove it and use the widow title bar to indicate where you are should you forget :)
